jenkins/deployment/k8s_deployment/cortx-cluster.groovy

```diff
@@ -0,0 +1,143 @@
+pipeline {
+    agent {
+        node {
+            label 'docker-centos-7.9.2009-node'
+        }
+    }
+    
+    options {
+        timeout(time: 600, unit: 'MINUTES')
+        timestamps()
+        buildDiscarder(logRotator(daysToKeepStr: '20', numToKeepStr: '20'))
+        disableConcurrentBuilds()
+    }
+    environment {
+        CORTX_RE_BRANCH = "kubernetes"
+        CORTX_RE_REPO = "https://github.com/Seagate/cortx-re/"
+        DOCKER_IMAGE_LOCATION = "https://github.com/Seagate/cortx-re/pkgs/container/cortx-all"
```

Comment:
Do we need this var?

Reply:
Yes. As of now, it's only supporting the GitHub registry but we need to extend it for the local registry.

Reply:
@shailesh-vaidya, could you please comment here that we need to make this var dynamic in future based on either github or local registry?

Reply:
Added comments